durablefunctionsmonitor.dotnetbackend/Common/Auth.cs

```diff
@@ -90,21 +90,23 @@ public static async Task ValidateIdentityAsync(ClaimsPrincipal principal, IHeade
                 throw new UnauthorizedAccessException($"'{DfmEndpoint.Settings.UserNameClaimName}' claim is missing in the incoming identity. Call is rejected.");
             }
 
-            if (DfmEndpoint.Settings.AllowedUserNames != null)
```

Comment:
Looks like this validation (checking whether user *name* is whitelisted) was dropped unintentionally.
It still needs to be here.

Apparently this is the reason why `ReturnsUnauthorizedResultIfUserNotWhitelisted` unit test is now failing. Another proof that unit tests are super useful.